docs/concepts/fractions.adoc

```diff
@@ -0,0 +1,32 @@
+[#fractions]
+= Fractions
+
+WildFly Swarm is defined by an unbounded set of capabilities. 
```

Comment:
Can you please explain this sentence? Does it mean that it does whatever you want it to?